rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -236,6 +236,9 @@ def get_positions(
             if status is True:
                 try:
                     trove_info = contract.decode(result, 'Troves', arguments=[addresses[idx]])
+                    trove_is_active = bool(trove_info[3])  # pylint: disable=unsubscriptable-object
```

Comment:
What is trove_info[3] supposed to be? I don't think the bool converter can raise in anyway but just wondering

Reply:
is either `0` or `1`